appveyor.yml

```diff
@@ -24,7 +24,10 @@ build_script:
   - conda update conda
   - conda install conda-build
   - ps: if(-not $env:APPVEYOR_PULL_REQUEST_NUMBER) { conda install anaconda-client }
-  - conda build conda.recipe --channel dynd/channel/dev
+  - conda build conda\ndt.recipe --channel dynd/channel/dev
+  - rd /q /s build dist dynd.ndt.egg-info
+  - echo dynd.ndt >> requirements.txt
```

Comment:
Hmm, the redirection works here (I looked at https://www.microsoft.com/resources/documentation/windows/xp/all/proddocs/en-us/redirection.mspx?mfr=true):

```
C:\Users\stefan\dynd-python>type requirements.txt
numpy

C:\Users\stefan\dynd-python>echo dynd.ndt >> requirements.txt

C:\Users\stefan\dynd-python>type requirements.txt
numpy
dynd.ndt
```

Yes, we can add ```dynd.ndt``` to ```install_requires``` in setup.py, depending on the target.